v3/spring-cloud-aws-v3-autoconfigure/src/main/java/io/awspring/cloud/v3/autoconfigure/properties/AwsRegionProperties.java

```diff
@@ -54,4 +55,40 @@ public void setStatic(String staticRegion) {
 		this.staticRegion = staticRegion;
 	}
 
+	public Profile getProfile() {
+		return profile;
+	}
+
+	public void setProfile(Profile profile) {
+		this.profile = profile;
+	}
+
+	public String getStatic() {
+		return this.staticRegion;
+	}
+
+	public static class Profile {
```

Comment:
We have same inner class `Profile` in `AwsCredentialsProperties` also. 
Should we extract it into separate class? If yes, to which package?

Reply:
Good point, lets extract it to `io.awspring.cloud.v3.autoconfigure.properties`.